index.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html>
+<html lang="es">
```

Comment:
That should be `en` for English, not `es` for Spanish, no?

Reply:
Yes. Fixed